app/src/main/java/com/marat/hvatit/playlistmaker2/presentation/medialibrary/MedialibraryViewPagerAdapter.kt

```diff
@@ -0,0 +1,24 @@
+package com.marat.hvatit.playlistmaker2.presentation.medialibrary
+
+import android.util.Log
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.lifecycle.Lifecycle
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+
+class MedialibraryViewPagerAdapter(fragmentManager: FragmentManager, lifecycle: Lifecycle) :
+    FragmentStateAdapter(fragmentManager, lifecycle) {
+    override fun getItemCount(): Int {
+        return 2
+    }
+
+    override fun createFragment(position: Int): Fragment {
+        Log.e("viewPagerMedialib", "i m here")
+        return when (position) {
+            0 -> FeaturedTracksFragment.newInstance("str")
+            1 -> PlaylistsFragment.newInstance("Android")
+            else -> PlaylistsFragment.newInstance("str")
```

Comment:
```suggestion
            0 -> FeaturedTracksFragment.newInstance()
            else -> PlaylistsFragment.newInstance()
```
Как аргументы удалишь - лучше сделать так